README.md

```diff
@@ -7,3 +7,18 @@ We are currently alpha testing this plugin.
 After installing the plugin (see [Installing Plugins](http://docs.grafana.org/plugins/installation/)), you will need to reach out to your LightStep representative to enable your account for alpha access and generate the API key needed in the configuration screen.
 
 Follow the configuration screen instructions to configure the plugin.
+
+## Templating
+See the [Templating](https://grafana.com/docs/grafana/latest/reference/templating/) documentation for an introduction to the templating feature and the different types of template variables.
+
+### Query variable
+The LightStep data source provides the following queries that you can specify in the `Query` field in the Variable edit view.
+
+| Name         | Description |
+| ------------ |-------------| 
+| `attributes(name)` <br/>`attributes(query)`    | Returns the Name of all Streams. This is can be used with the `Regex` field in Grafana to build a dropdown |
+| `stream_ids(query=~"regex")` <br/>`stream_ids(query!=~"regex")` <br/>`stream_ids(query="value")` <br/>`stream_ids(query!="value")`    | Returns the Stream ID of all matching Streams. The datasource uses the Stream ID to request the timeseries data in the various panel/visualization. |
```

Comment:
I see in the screenshots provided a query example like `stream_ids(name="something something")`. Please document an example like that in this table.